lively.morphic/morph.js

```diff
@@ -1641,7 +1641,8 @@ export class Morph {
     return this;
   }
 
-  delete () {
+  abandon () {
+    // We do not have access to the JS VM, this method does not interact with the garbage collector and may not result in the actual deletion of the object. However, when an object is abandoned, no properties are guaranteed.
```

Comment:
Maybe change to: `// Use this method to signal the wish to permanently delete this object. Overwrite this method to clean up resources on its deletion. We do not have access to the JS VM, this method does not interact with the garbage collector and does not result in the actual deletion of the object, if there are still left over references to this object.`